_posts/2019-06-18-list-strategy.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: posts
+title: "Automatic execution strategy selection in Bazel 0.27"
+authors:
+  - ishikhman
+---
+**tl;dr**: Exciting news! After 0.27 release Bazel will *auto select* a suitable execution strategy if none is provided. Customization and strategy enforcement will still be possible and will be much easier to configure.
```

Comment:
nit. add link to https://blog.bazel.build/2019/06/17/bazel-0.27.0.html

Reply:
Also: do you want to update the blog post to include a link to this page?

Reply:
Good idea, I will, ones this page is published